### ISSUE TITLE
Need additional properties in the dcmi:Collection file?

### ISSUE THREAD

**Issue by briesenberg07** [UniverityOfWashingtonLibrariesSchemas/schemasProject#7]
Physical and digital collections appear the same in the collection file. dpla:SourceResources link to dcmi:Collections via dct:isPartOf regardless of whether collections are digital or physical. Additionally, while the AYP digital collection has dct:title "Alaska-Yukon-Pacific Exposition Collection", there is also a physical collection with this same name.  
***So, should we use additional properties in the Collection file to differentiate between digital and physical collections?***
The DPLA MAP uses both dct:description and edm:isShownAt to describe dcmi:Collection resources. We could use the former for physical collections and both for digital collections?

**Comment by briesenberg07** [comment]
Sorry, I think I understand now.  
Physical collections are linked to from source resources, digital collections are linked to from web resources.  
This would make differentiating between them in the agents file unnecessary. Unless there is a hypothetical situation where someone is only using the collection file?

**Comment by gerontakos** [comment]
Right, we don’t need to differentiate, there is already differentiation. However we have the “files” partly because we want to have the option of adding to them as we see fit. The Collections file would make more sense if it was one continuous collection dataset for all UW Libraries collections. So as we go on – but early in the process – we should probably combine all “collections” files.

So we could add whatever we want to the collection dataset.

What I think we do need to do there is to have only one URI for each entity. Right now some entities, or collections in this case, have multiple URIs:
http://doi.org/10.6069/uwlib.55.A.3.4#Alaska-Yukon-PacificExpositionCollection
http://doi.org/10.6069/uwlib.55.A.3.4#Alaska-Yukon-Pacific-ExpositionCollection

The hard part about this is that the URIs cleanes in the collection dataset also need to be cleaned in the sourceresource and webresource files.

  --Theo

---------------------------------------------------------
nameOfSender=Theodore_Gerontakos
jobTitle=Metadata&#47;Cataloging_Librarian
cellPhone=206-276-6209
officePhone=206-543-8407
---------------------------------------------------------

**Comment by briesenberg07** [comment]
### Note:
Based on XML metadata, these are actually two entities:
- **DigitalCollection:** Alaska-Yukon-Pacific Exposition Collection (for all 1425 items, with variant "Alaska-Yukon-Pacific-Exposition Collection")
- **RepositoryCollection:** Alaska-Yukon-Pacific Exposition Collection (for 38 items)
